dot/core/service_test.go

```diff
@@ -35,19 +35,22 @@ import (
 	"github.com/ChainSafe/gossamer/lib/transaction"
 	"github.com/ChainSafe/gossamer/lib/trie"
 	"github.com/ChainSafe/gossamer/tests"
+	"github.com/stretchr/testify/require"
 )
 
-var TestMessageTimeout = 5 * time.Second
+// TestMessageTimeout is the wait time for messages to be exchanged
+var TestMessageTimeout = time.Second
```

Comment:
Why has this changed from 5 seconds to 1?

Reply:
the messages are exchanged within milliseconds and sometime when we wait 5 seconds nodes connect and disconnect several times and exchange messages more than we need